packages/api-client/src/test/node/spec/Client.test.js

```diff
@@ -128,43 +128,50 @@ describe('Client', () => {
       },
     ];
 
+    const cookie_zuid_login =
+      'SBMt4gP7v-SAxxP_8OEVVeTw11CeUBAV1Jx5AwdQNjcbgTIqvkhfmd8COLG5V3OrOJ==.v=1.k=1.d=1535117120.t=1234';
+    const cookie_zuid_refresh =
+      'Xfnt4gP7v-SAxxP_8OEVVeTw110eUBAV1Jx5bwdQNjcbgTIqvhhfmd8CBLG5V3OrOJ==.v=1.k=1.d=2535100120.t=5678';
+    const cookie_login = `zuid=${cookie_zuid_login}; Path=/access; Domain=example.com; HttpOnly; Secure`;
+    const cookie_refresh = `zuid=${cookie_zuid_refresh}; Path=/access; Domain=example.com; HttpOnly; Secure`;
+
     beforeEach(() => {
       nock(baseURL)
         .post(`${AuthAPI.URL.LOGIN}`, {
           email: loginData.email,
           password: loginData.password,
         })
-        .query({persist: loginData.clientType === 'permanent'})
-        .reply(200, accessTokenData);
+        .query(true)
+        .reply(200, accessTokenData, {
+          'set-cookie': cookie_login,
+        });
 
       nock(baseURL)
         .post(`${AuthAPI.URL.ACCESS}/${AuthAPI.URL.LOGOUT}`)
-        .reply(200, undefined);
+        .reply(200);
+    });
+
+    afterEach(() => {
+      nock.cleanAll();
     });
 
-    it('creates a context from a successful login', done => {
+    it('creates a context from a successful login', () => {
       const client = new APIClient();
-      client.login(loginData).then(context => {
+      return client.login(loginData).then(context => {
         expect(context.userId).toBe(accessTokenData.user);
         expect(client.accessTokenStore.accessToken.access_token).toBe(accessTokenData.access_token);
-        done();
       });
     });
 
-    it('can login after a logout', done => {
+    it('can login after a logout', () => {
       const client = new APIClient();
-      client
-        .login(loginData)
-        .then(() => client.logout())
-        .then(done)
-        .catch(done.fail);
+      return client.login(loginData).then(() => client.logout());
```

Comment:
Isn't there an assertion we can make here?

Reply:
Also just reading the test, it doesn't seem we try to login after a logout but rather the other way around (trying to logout after a login). Am I wrong?

Reply:
Since it's not new code, not a blocker, but having a look at it could be cool :)